rasa_sdk/__init__.py

```diff
@@ -1,15 +1,14 @@
 import logging
 
+import rasa_sdk.cli
 import rasa_sdk.version
+from rasa_sdk.interfaces import Tracker, Action, ActionExecutionRejection  # noqa: F401
```

Comment:
I am not sure what you mean here. Can you explain?

Reply:
I do not see that these imports are used by the code below. That is why I think that some other parts of the imported code require these imports in `__init__.py`

Reply:
it's because they're part of the public "API" of the SDK. We want people to be able to do `from rasa_sdk import ...`, for ease of use. I wouldn't change it